src/main/java/org/broadinstitute/hellbender/engine/dataflow/coders/GATKReadCoder.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.engine.dataflow.coders;
+
+
+import com.google.cloud.dataflow.sdk.coders.CoderException;
+import com.google.cloud.dataflow.sdk.coders.CustomCoder;
+import com.google.cloud.dataflow.sdk.coders.SerializableCoder;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.GoogleGenomicsReadToGATKReadAdapter;
+import org.broadinstitute.hellbender.utils.read.SAMRecordToGATKReadAdapter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class GATKReadCoder<T extends GATKRead> extends CustomCoder<T> {
```

Comment:
add class comment
Is this class intendend for subclassing? If yes, describe what is the contract for the subclass, ie what do subclasses need to do. If it's not intended for subclassing, make it final.